README.md

```diff
@@ -23,3 +20,20 @@ metadata:
     aws.com/lambda/function-name: HelloWorld
     aws.com/lambda/region: us-east-1
 ```
+
+In order to add overview widget to your backstage entity page, please update
+
+```js
+// packages/app/src/components/catalog/EntityPage.tsx
+
+import { AWSLambdaOverviewWidget } from '@roadiehq/backstage-plugin-aws-lambda';
+
+const OverviewContent = ({ entity }: { entity: Entity }) => (
+  <Grid container spacing={3}>
+    ...
+    <Grid item>
+      <AWSLambdaOverviewWidget entity={entity} />
```

Comment:
Does the router export an isPluginApplicabletoEntity function ? worth documenting that I think. In this case where all components won't be lambdas users might want to hide that Grid element if the plugin isn't applicable (we explain the interface and let them use it as they see fit)

Reply:
typescript highlights that you can import this function. But API documentation would be also nice. If we decide on it, I will also write it for travis-ci and firebase-functions plugins.

Reply:
This is true, just commited this change @iain-b